tests/para-tests/moonbase/test-eth-rpc-block-tags.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from "chai";
+import { ALITH_ADDRESS, ALITH_GENESIS_TRANSFERABLE_BALANCE } from "../../util/accounts";
+
+import { describeParachain } from "../../util/setup-para-tests";
+
+describeParachain(
+  "Ethereum RPC block tags",
+  {
+    parachain: {
+      chain: "moonbase-local",
+    },
+  },
+  (context) => {
+    it("should support pending tag", async function () {
+      this.timeout(150000);
+      await context.waitBlocks(1);
+      const expectedBalance = await context.web3.eth.getBalance(ALITH_ADDRESS, "pending");
+      expect(BigInt(expectedBalance)).to.equal(ALITH_GENESIS_TRANSFERABLE_BALANCE);
+    });
+    // Un-skip when we pull latest frontier support for PoS `safe` and `finalized` tags.
+    // Also bump web3 to at least 1.8.0-rc.0.
+    it.skip("should support merge tags in the parachain context", async function () {
```

Comment:
needs to remove the `skip`
Otherwise it is good

Reply:
It's skipped on purpose - as the new `safe` and `finalized` tags will be introduced once we do the upcoming `0.9.29`. I added already the tests, so we can just unskip it when it's done.